src/Selenium2Library/keywords/_selectelement.py

```diff
@@ -8,15 +8,24 @@ class _SelectElementKeywords(KeywordGroup):
 
     # Public
 
-    def get_list_items(self, locator):
-        """Returns the values in the select list identified by `locator`.
+    def get_list_items(self, locator, label=True):
+        """Returns the labels or values in the select list identified by `locator`.
 
         Select list keywords work on both lists and combo boxes. Key attributes for
         select lists are `id` and `name`. See `introduction` for details about
         locating elements.
+        
+        Sample:
+        | ${labels1} = | Get List Items | xpath=//h1 |
+        | ${labels2} = | Get List Items | xpath=//h1 | label=${True} |
+        | ${values} = | Get List Items | xpath=//h1 | label=${False} |
+        | Should Be Equal | ${labels1} | ${labels2} |
         """
         select, options = self._get_select_list_options(locator)
-        return self._get_labels_for_options(options)
+        if label:
```

Comment:
This will be true if someone executes this keyword like `label=False`. A workaround is using `label=${FALSE}` (as shown in the docs), but that's ugly. RF itself handles these cases so that strings `false` and `no`, case-insensitively, are considered false. Was there @aaltat already an issue about handling Boolean arguments consistently within the library and, preferably, same way as Robot handles them? If there is, perhaps this could just be added there as one place to fix later.

Alternatively the signature could be changed to `value=False`. They you'd basically never wanted to pass explicit false value and using `value=True` on Robot would work.